### ISSUE TITLE
doc is still null in 3.1.2

### ISSUE THREAD

**Issue by charlottedamm** [sanity-io/sanity-plugin-iframe-pane#91]
**Describe the bug**

The bug where the the document property for the url option is null when there's no draft version of a document, is still there after the 3.1.2. update

**To Reproduce**

See issues https://github.com/sanity-io/sanity-plugin-iframe-pane/issues/89 and https://github.com/sanity-io/sanity-plugin-iframe-pane/issues/88

**Which versions of Sanity are you using?**

```
@sanity/cli (global)          3.20.1 (up to date)
@sanity/assist                1.2.15 (up to date)
@sanity/color-input            3.1.0 (up to date)
@sanity/eslint-config-studio   3.0.1 (up to date)
@sanity/ui                     1.9.3 (up to date)
@sanity/vision                3.20.1 (up to date)
sanity                        3.20.1 (up to date)
```

**Which versions of Node.js / npm are you running?**

```
v20.9.0
10.1.0
```

**Comment by rdunk** [comment]
Closing in favour of reopening #89.

**Comment by salex97** [comment]
@rdunk as #89 is locked ("This conversation has been locked as resolved and limited to collaborators."), I'll answer your question here:
> Are you able to confirm if 3.1.2 fixes the issue for you?

Yes, for me the issue is fixed by 3.1.2, everything works fine now.
Thx for the quick fix!

**Comment by rdunk** [comment]
Pesky bot, sorry about that. Conversation is now unlocked.

Thanks for confirming @salex97.

**Comment by rdunk** [comment]
#89 was closed by the author so unlocking the conversation here.

The issue has been reported as fixed by others, @charlottedamm would you be able to confirm that you are definitely experiencing this problem using `3.1.2`? Can reopen this issue if so.

**Comment by charlottedamm** [comment]
@rdunk yes, still experiencing this issue with 3.1.2, both locally and in production

**Comment by rdunk** [comment]
Thanks for confirming, fix incoming in #92.
